### DISCUSSION TITLE
Prevent audio download

### DISCUSSION THREAD

**Discussion by mohamedhafizelhaj** [symfony/symfony#47501] [Q&A]
I am using ```BinaryFileResponse``` to send an audio to the client in chunks. I am setting the ```content-disposition``` to ```inline```, I want to prevent the audio download from third-party applications like IDM, how can I do that ?

**Comment by MatTheCat** [comment]
I don't really know what you mean by “prevent” 🤔

Do you mean rejecting requests without a `Range` header? Or restricting media access in your application only? Or something else?

The first case would be easy to implement in the controller, and I guess the second would require [EME](https://web.dev/eme-basics/).
